fix (#3)
scalar-labs/scalardl-client-sdk: docs/dl-getting-started.md

In Scalar DL 1.0, `asset_id` is not a reserved json key name and you can use any json key name, but this will probably be changed in future versions.
(`asset_id` and `asset_ids` will be reserved for future enhancements.)

> ## Create your contracts

Reviewer: There is already a "Create a contract" section above. I think the point of this section is the link to the emulator so perhaps it should be renamed to "Scalar DL Emulator" and expanded from there.

Author: My intention is not only for Emulator. 
I feel we also want to say "create your contract" instead of creating what's given.
Let me revise the sentence and the section title.

Reviewer: Ah, I see. Maybe then the section title could be "Create your own contracts" or something?

> As we explained above, what you need to do is implement the `invoke` method as you like.

Reviewer: I don't think that this has actually been explained above...

Probably this line and the next should be deleted.

We are preparing more sample contracts, so please wait for an update.

To quickly run and test your contrats, [Scalar DL Emulator](https://github.com/scalar-labs/scalardl-emulator) might be useful.

> It uses mutable in-memory ledger instead of immutable ledger database and provides interactive interface, so you can do trial and errors more easily.

Reviewer 2: ```suggestion
It uses a mutable in-memory ledger instead of an immutable ledger database and provides an interactive interface, making it easy to do trial and error testing.
```

In Scalar DL 1.0, `asset_id` is not a reserved json key name and you can use any json key name, but this will probably be changed in future versions.
(`asset_id` and `asset_ids` will be reserved for future enhancements.)

## Create your contracts

> It's about time to create your contracts.

Reviewer 2: ```suggestion

```

> As we explained above, what you need to do is implementing `invoke` method as you like.

Reviewer 2: ```suggestion
As we explained above, what you need to do is implement the `invoke` method as you like.
```
